### PR TITLE
fix(docs-infra): removal of the use of the ChildNode.remove() method …

### LINKED ISSUE #28177: angular.io doesn't work with IE11
Hi,

I have tested angular.io on IE 11 and it doesn't work. The first page is loaded but we have this exception:

```
"TypeError: Objet attendu\n   at e.prototype.reportError (https://angular.io/main.09223f95b24b4723adce.js:1:493566)\n   at e.prototype.handleError (https://angular.io/main.09223f95b24b4723adce.js:1:493467)\n   at next (https://angular.io/main.09223f95b24b4723adce.js:1:65566)\n   at Anonymous function (https://angular.io/main.09223f95b24b4723adce.js:1:46907)\n   at e.prototype.__tryOrUnsub (https://angular.io/main.09223f95b24b4723adce.js:1:147806)\n   at e.prototype.next (https://angular.io/main.09223f95b24b4723adce.js:1:146948)\n   at e.prototype._next (https://angular.io/main.09223f95b24b4723adce.js:1:145958)\n   at e.prototype.next (https://angular.io/main.09223f95b24b4723adce.js:1:145641)\n   at e.prototype.next (https://angular.io/main.09223f95b24b4723adce.js:1:173361)\n   at e.prototype.emit (https://angular.io/main.09223f95b24b4723adce.js:1:46673)"


{
   [functions]: ,
   __proto__: { },
   description: "Objet attendu",
   message: "Objet attendu",
   name: "TypeError",
   number: -2146823281,
   stack: "TypeError: Objet attendu
   at e.prototype.reportError (https://angular.io/main.09223f95b24b4723adce.js:1:493566)
   at e.prototype.handleError (https://angular.io/main.09223f95b24b4723adce.js:1:493467)
   at t.prototype.error (https://angular.io/main.09223f95b24b4723adce.js:1:397062)
   at Anonymous function (https://angular.io/main.09223f95b24b4723adce.js:1:461855)
   at e.prototype.error (https://angular.io/main.09223f95b24b4723adce.js:1:13724)
   at e.prototype._error (https://angular.io/main.09223f95b24b4723adce.js:1:405778)
   at e.prototype.error (https://angular.io/main.09223f95b24b4723adce.js:1:145721)
   at e.prototype.notifyError (https://angular.io/main.09223f95b24b4723adce.js:1:176557)
   at e.prototype._error (https://angular.io/main.09223f95b24b4723adce.js:1:375068)
   at e.prototype.error (https://angular.io/main.09223f95b24b4723adce.js:1:145721)"
}
```

And when I click on an other link, we have this error:

```

ERROR Error: [DocViewer] Error preparing document 'guide/component-interaction': TypeError: L’objet ne gère pas la propriété ou la méthode « remove »
```

 Version: IE 11.0.9600.19129

### PR DESCRIPTION
**Pull request by wKoza**
…that it isn't supported by IE

fixes #28177

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Removing the use of ChildNode.remove() that it isn't supported by IE.

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

angular.io doesn't work with IE 

Issue Number: #28177

## What is the new behavior?

angular.io works with IE ?

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

@gkalpak , can you call Mary Poppins for me ? I can't test it at home.

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
I approved this but I have not checked it on IE11.

**Comment by wKoza** [comment]
I'll try it tomorrow

**Comment by wKoza** [comment]
It's OK! We can merge it !

**Review by gkalpak — APPROVED** [review]
👌 

Another issue I noticed is that our SVG `mat-icon`s do not appear on IE11 (which sucks), but I couldn't find out why. I thought it might be because we use [self-closing <path />][1], but apparently not.
Any ideas, @wKoza?

[1]: https://github.com/angular/angular/blob/8a08ff1571bae6e3a4a6bb46006edcbddda46255/aio/src/app/app.module.ts#L90

**Comment by wKoza** [comment]
Yes @gkalpak indeed, there are a lot of display errors with svg. And the map is weird, it doesn't miss something (also with Chrome) ?
capture
I'll try to found out why...

File: aio/src/app/layout/doc-viewer/doc-viewer.component.ts [hunk 1]
```diff
@@ -100,9 +100,9 @@ export class DocViewerComponent implements OnDestroy {
     if (needsToc && !embeddedToc) {
       // Add an embedded ToC if it's needed and there isn't one in the content already.
       titleEl!.insertAdjacentHTML('afterend', '<aio-toc class="embedded"></aio-toc>');
-    } else if (!needsToc && embeddedToc) {
+    } else if (!needsToc && embeddedToc && embeddedToc.parentNode !== null) {
       // Remove the embedded Toc if it's there and not needed.
-      embeddedToc.remove();
+      embeddedToc.parentNode.removeChild(embeddedToc);
```
[the remark this answers is not in the document]
Comment (alfaproject) [reply aio/src/app/layout/doc-viewer/doc-viewer.component.ts]: It looks like someone needs to come up with a TSLint ES11 ruleset, lol

[the remark this answers is not in the document]
Comment (wKoza) [reply aio/src/app/layout/doc-viewer/doc-viewer.component.ts]: done

**Comment by wKoza** [comment]
I think to have found out for the svg 'GET STARTED' but I have no idea about the mat-icon. We should open a new issue for that.

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**